tests/e2e/macvtap_test.go

```diff
@@ -81,10 +81,10 @@ var _ = Describe("macvtap-cni", func() {
 			})
 
 			It("THEN a macvtap custom network resource is exposed", func() {
-				quantity := 50
+				quantity := 100
```

Comment:
Because 100 is the default capacity. Previously when using a explicit config, the capacity was set at 50. Now that we are using default config, default capacity is 100.